src/ui/validation.js

```diff
@@ -25,6 +27,22 @@ export function validateUrl(url, t, { allowInsecure = false } = {}) {
   return { validateStatus, help };
 }
 
+export function validateMarkdown(markdown, t) {
+  let validateStatus;
+  let help;
+
+  if (MARKDOWN_REGEX_BOLD_OR_ITALIC_WITHIN_HEADERS.test(markdown)) {
```

Comment:
Maybe we should mark in the ticket the caveat that this method does not warn the user if they use the alternative way to define headers.